lua/entities/gmod_wire_expression2/core/custom/prop.lua

```diff
@@ -628,7 +628,7 @@ end
 __e2setcost(20)
 e2function void entity:setPos(vector pos)
 	if not ValidAction(self, this, "pos") then return end
-	PhysManipulate(this, pos, nil, nil, nil, nil)
+	WireLib.setPos(this, pos)
```

Comment:
Can you use the locals `setPos` and `setAng` instead? It's already defined so you may as well use them. See line 16.